packages/puppeteer-extra-plugin/package.json

```diff
@@ -64,7 +64,7 @@
   "dependencies": {
     "@types/debug": "^4.1.0",
     "debug": "^4.1.1",
-    "merge-deep": "^3.0.1"
+    "deepmerge": "^4.2.2"
```

Comment:
Same quetsion as above.

Reply:
@jozsi We are using it here https://github.com/berstend/puppeteer-extra/blob/d5d82b4506a6ca514b7852709f30a7815f8a1304/packages/puppeteer-extra-plugin/src/index.ts#L5